app/assets/stylesheets/styles/guides/_new_guides.css.scss

```diff
@@ -43,111 +134,137 @@
       border-bottom-right-radius: $global-radius;
     }
 
-    .step{
+    .step {
       padding: .5rem 1rem;
       border: 1px $secondary-color solid;
     }
 
-    .step.active{
+    .step.active {
       background-color: $of-dark;
       border: 1px $of-dark solid;
       color: white;
     }
 
-    .preview{
+    .preview {
       border-radius: $global-radius;
       opacity: .5;
     }
 
-    .clickable{
+    .clickable {
```

Comment:
I'm hoping you just did find & replace on this, though I'm totally picturing you going through these one by one and fixing them.